packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -1,3 +1,7 @@
+import {
+  ConnectWalletParams,
+  SignMessageParams,
+} from '@notifi-network/notifi-core';
```

Comment:
ah actually I wanted to deprecate `notifi-core` since it's a little uncomfortable to use.

The ideal developer experience should be that I can import all the types I need from the package that I depend on (i.e. if I'm only using react-card, I should be able to import everything from it)

For now, I think it's okay because it helps us to DRY things